src/main/scala/gemmini/ReservationStation.scala

```diff
@@ -54,7 +54,7 @@ class ReservationStation[T <: Data : Arithmetic, U <: Data, V <: Data](config: G
   })
 
   // TODO make this a ChiselEnum
-  val ldq :: stq :: exq :: Nil = Enum(3)
+  val ldq :: exq :: stq :: Nil = Enum(3)
```

Comment:
Why did you change the order?

I'm fine with the new order; I'm just curious

Reply:
Initially I concatenated the separated lists this way without paying attention the enum order, then the issue id's started to depend on the enum order as well. Between reconcatenating all the lists and changing the enum I chose the latter. As for the initial decision, I thought it made a little more sense because this is usually the workflow (load, use it, then store the result).

Reply:
Sounds good; just add a comment saying that the later code assumes this ordering